core/jazz_metrics/index.js

```diff
@@ -49,7 +49,7 @@ function handler(event, context, cb) {
     exportable.genericValidation(event)
       .then(() => validateUtils.validateGeneralFields(eventBody))
       .then(() => exportable.getToken(config))
-      .then((authToken) => exportable.getAssetsDetails(config, eventBody, authToken))
+      .then((authToken) => exportable.getAssetsDetails(config, eventBody, authToken, event.headers['Jazz-Service-ID']))
```

Comment:
relevant update to getAssetsDetails function ?

Reply:
Done